src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -23,6 +30,37 @@
         (ds/rename-columns {col-to-avg avg-name})
         (wds/rollup :mean avg-name [:gss-code :sex :age]))))
 
+(defn make-trend-key [key]
+  (keyword (str/join [(name key) "-trend"])))
```

Comment:
This function could now be dropped or used to name the new column output from either of these jumpoffyr functions ??